processors/config/anaTrackTuple_cfg.py

```diff
@@ -0,0 +1,36 @@
+import HpstrConf
+import sys
+
+# Use the input file to set the output file name
+inFilename  = sys.argv[1].strip()
+outFilename = '%s_anaTrks.root' % inFilename[:-5]
+
+print('Input file:  %s' % inFilename)
+print('Output file: %s' % outFilename)
+
+p = HpstrConf.Process()
+
+# Library containing processors
+p.libraries.append("libprocessors.so")
```

Comment:
Hi Maurik,
Thank you for reviewing the pr.
Oh, thanks! That's cool, I forgot you fixed this naming convention. We should change this. I'll do a commit with this change and add it to the iss26 branch.

Reply:
Regarding the AnaProcessors.. They are in the processors because they are actually part of the libProcessors they extend the processor base class. That's why they are there.
Perhaps one folder per analysis is too much, not sure.. hard to predict right now how much this is going to grow... 

Cheers,
PF

Reply:
You can extend a class that is loaded from a (different) library. It just seems a better way to organize the code.
Also, it will be an example of how to start your own directory with your own analysis, that you may not want to mingle with the official code. I can imagine that especially for students it would be less intimidating if they worked in their own directory.

Reply:
I thought the entire idea of this project was to keep people from doing exactly this. We want people to keep extending this until it is capable of all the analysis tasks we need to achieve. Students can make their own directory with a set of python config files to develop their own analysis via changing these config files. We should avoid everyone having personal versions of the processors, as this is a collaborative effort. We should not have an analysis directory for every analyst.